l1-contracts/test/foundry/l1/integration/l2-tests-in-l1-context/L2GatewayTestAbstract.t.sol

```diff
@@ -39,32 +39,25 @@ abstract contract L2GatewayTestAbstract is Test, SharedL2ContractDeployer {
         finalizeDeposit();
         require(l2Bridgehub.ctmAssetIdFromAddress(address(chainTypeManager)) == ctmAssetId, "ctmAssetId mismatch");
         require(l2Bridgehub.ctmAssetIdFromChainId(mintChainId) == ctmAssetId, "ctmAssetIdFromChainId mismatch");
-
         address diamondProxy = l2Bridgehub.getZKChain(mintChainId);
         require(!GettersFacet(diamondProxy).isPriorityQueueActive(), "Priority queue must not be active");
     }
-
```

Comment:
the changes here while do not change the real logic, they would still make the file less readable IMO (especially removal of the newlines between functions).

I would suggest to just keep the file as it is

Reply:
Reverted them back to the earlier state, don't know how this change happened.